Changes.md

```diff
@@ -1,6 +1,16 @@
 0.61.x.x (relative to 0.61.13.1)
 =========
 
+Improvements
+------------
+
+- PathListingWidget :
+  - Added an option to `selectionMode` allowing single and multiple cell selections.
+  - Added the ability to select cells in different columns with the left <kbd>&larr;</kbd> and right <kbd>&rarr;</kbd> keys.
+  - Added the ability to drag from a cell and drop that cell's value on an appropriate target.
+  - Improved the range selection behavior when using <kbd>Shift</kbd> + arrow keys. It now selects cells in a similar way to <kbd>Shift</kbd> + Click.
```

Comment:
I don't think this has changed from a user perspective - the old rows-only behaviour is the same as before, and the cell modes are brand new.